### PR TITLE
WRQ-24303: Add marqueeController with content changed samples in marquee qa-sampler

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Relates to https://github.com/enactjs/enact/pull/3247 PR, I added marquee qa samples to test 
changing contents of marquee which is used with marqueeController in this PR.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Add samples

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-24303

### CODE REVIEW

File: samples/sampler/stories/qa/Marquee.js [hunk 1]
```diff
@@ -168,6 +168,39 @@ class MarqueeWithContentChanged extends Component {
 	}
 }
 
+class MarqueeSynchronizedWithContentChanged extends Component {
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Marquee.js:171]: Thanks for adding this!

File: samples/sampler/stories/qa/Marquee.js [hunk 2]
```diff
@@ -168,6 +168,39 @@ class MarqueeWithContentChanged extends Component {
...
+				<ol>
+					<li>Click once to show the ellipsis just before the text marquees the first time.</li>
+					<li>
+						Click a second time to show the ellipsis just before the text marquees the first time
+					</li>
+					<li>Click again to return to a short string without marquee.</li>
+				</ol>
+				<Button onClick={this.handleClick}>{'Click Me'}</Button>
+				<Controller>
+					<Marquee style={{width: '800px'}} marqueeOn={'render'}>
```
Comment (MikyungKim) [inline samples/sampler/stories/qa/Marquee.js:195]: Let's use `ri.ScaleToRem` for the number.

Comment (juwonjeong) [reply samples/sampler/stories/qa/Marquee.js]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**